Hestia/app/src/androidTest/java/com/rugged/application/hestia/backend/NetworkHandlerTest.java

```diff
@@ -64,23 +73,23 @@ public void getDefaultPathTest() {
 
     @Test
     public void equalsTest() {
-        NetworkHandler handler1 = this.dummyHandler;
-        assertNotNull(handler1);
-        assertTrue(dummyHandler.equals(handler1));
+        NetworkHandler handlerCurrent = this.dummyHandler;
+        assertNotNull(handlerCurrent);
+        assertTrue(dummyHandler.equals(handlerCurrent));
 
-        NetworkHandler handler2 = new NetworkHandler(dummyIP, dummyPort);
-        assertNotNull(handler2);
-        assertTrue(dummyHandler.equals(handler2));
+        NetworkHandler handlerSameProperties = new NetworkHandler(dummyIP, dummyPort);
+        assertNotNull(handlerSameProperties);
+        assertTrue(dummyHandler.equals(handlerSameProperties));
 
-        NetworkHandler handler3 = null;
-        assertNull(handler3);
-        assertFalse(dummyHandler.equals(handler3));
+        NetworkHandler handlerNull = null;
+        assertNull(handlerNull);
+        assertFalse(dummyHandler.equals(handlerNull));
 
         String IP = "1.1.1.1";
         Integer port = 2000;
-        NetworkHandler handler4 = new NetworkHandler(IP, port);
-        assertNotNull(handler4);
-        assertFalse(dummyHandler.equals(handler4));
+        NetworkHandler handlerDifferentProperties = new NetworkHandler(IP, port);
+        assertNotNull(handlerDifferentProperties);
+        assertFalse(dummyHandler.equals(handlerDifferentProperties));
     }
```

Comment:
This could  be divided into multiple methods, this makes it easier to see which assert fails.